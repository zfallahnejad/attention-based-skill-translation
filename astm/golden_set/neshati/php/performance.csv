138475,performance
187606,performance
338665,performance
476,performance
324584,performance
180136,performance
50079,performance
251311,performance
18393,performance
797303,performance
4833,performance
