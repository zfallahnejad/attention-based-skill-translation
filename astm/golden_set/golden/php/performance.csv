138475,13,13,0.5
187606,13,18,0.41935483870967744
338665,12,9,0.5714285714285714
324584,11,14,0.44
476,11,10,0.5238095238095238
50079,9,5,0.6428571428571429
180136,9,2,0.8181818181818182
18393,8,11,0.42105263157894735
251311,8,7,0.5333333333333333
4833,7,6,0.5384615384615384
797303,7,9,0.4375
